src/index.css

```diff
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+td,
+th {
+  border: 1px solid #ddd;
+  text-align: left;
+  padding: 8px;
+}
+
+tr:nth-child(even) {
+  background-color: #ddd;
+}
+
+.header {
+  background-color: darkslateblue;
+  color: #ddd;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.arrow {
+  color: white;
+}
+
+.coin-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #aed6f1;
+  gap: 20px;
+}
+
+.search-bar-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.search-bar {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+  border: 1px solid red;
+  border-radius: 4px;
+}
+
+.coin-topic {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: center;
+}
+
+.coin-card {
+  text-decoration: none;
+  color: white;
+  width: 50%;
+  display: flex;
+  align-items: center;
+  background-color: #eb984e;
```

Comment:
- [ ] As per the `design` guidelines you are allowed to use only one main colour for your application which in your case should be `#483d8b` not `#eb984e`, your application design should be consistent because of the multiple colors in your `website` it makes your app looks not good and also make your design looks inconsistent too. Kindly follow the `design` instructions properly. 

![Screenshot from 2023-11-25 03-27-30](https://github.com/tawakuliKH/coinCap-React/assets/66476329/c0884869-047d-4620-9c1e-e421af11cddf)